boot/boot_serial/src/serial_recovery_cbor_readme.txt

```diff
@@ -0,0 +1,16 @@
+The serial_recovery_cbor.c|h files have been generated with cddl_gen.py from the
+description in serial_recovery.cddl.
+cddl_gen.py has been copied from the "cddl_gen" repo, commit
```

Comment:
Please refer here to the original used cddl_gen repo (https://github.com/oyvindronningstad/cddl_gen) as there is a chance that cddl_gen may get forked a lot and this may cause confusion.